.gitignore

```diff
@@ -4,6 +4,7 @@ composer.phar
 clover.xml
 coveralls-upload.json
 phpunit.xml
+preload.php
```

Comment:
We WANT the preload file checked in. While it has rules to dynamically create the list of files, _it also has an API so you can decide whether or not to ignore files or classes, and/or add more paths_.